doc/source/ray-overview/basics.rst

```diff
@@ -0,0 +1,24 @@
+
+.. raw:: html
+
+  <embed>
+    <a href="https://github.com/ray-project/ray"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/365986a132ccd6a44c23a9169022c0b5c890c387/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f7265645f6161303030302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png"></a>
+  </embed>
+
+.. image:: https://github.com/ray-project/ray/raw/master/doc/source/images/ray_header_logo.png
+
+**Ray is a fast and simple framework for building and running distributed applications.**
+
+Ray accomplishes this mission by providing:
+
+1. Providing simple primitives for building and running distributed applications.
+2. Enabling end users to parallelize single machine code, with little to zero code changes.
+3. Including a large ecosystem of applications, libraries, and tools on top of the core Ray to enable complex applications. 
+
+Ray is organized into four key components:
```

Comment:
There are five. I'm not sure I like this wording, it doesn't really make it obvious that the latter four are just libraries written on Ray and we want to encourage others to write libraries as well

Reply:
Think I fixedx this appropriately, let me know if not.